bftengine/src/bftengine/ClientsManager.hpp

```diff
@@ -82,7 +82,7 @@ class ClientsManager : public ResPagesClient<ClientsManager, 0> {
   // General
   static uint32_t reservedPagesPerClient(const uint32_t& sizeOfReservedPage, const uint32_t& maxReplySize);
   int getIndexOfClient(const NodeIdType& id) const;
-
+  
```

Comment:
I guess you touched this file occasionally?

Reply:
I was about to make a new function that had to do the same thing as the one above. It seems that I've left a blank space unintentionally.